### PR TITLE
feat: build pages statically

### PR DESCRIPTION
**Pull request by backflip**
In order to increase reliability and performance we should statically build all pages.

This should fix issues like https://www.contentfulstatus.com/incidents/1ml52n7pbdv9 taking down the whole page.

### CODE REVIEW

File: src/pages/[slug].astro [hunk 1]
```diff
@@ -9,7 +29,7 @@ import Venue from "../components/Venue.astro";
 import HotelList from "../components/HotelList.astro";
 import RestaurantList from "../components/RestaurantList.astro";
 import AirtableForm from "../components/AirtableForm.astro";
-
+console.log(Astro.params);
```
Comment (uncenteredDiv) [inline src/pages/[slug].astro]: **suggestion**: guess this can be removed

Comment (backflip) [reply src/pages/[slug].astro]: Done

**Comment by uncenteredDiv** [comment]
LGTM!

The only thing I noticed, is that we are now building all the Speaker and Workshop Pages - even though they are not Part of this years Conference ... but since we don't have active links to them that should be ok (and we will need this for the Archive later anyway)

**Review by uncenteredDiv — APPROVED** [review]

**Comment by backflip** [comment]
> LGTM!
> 
> The only thing I noticed, is that we are now building all the Speaker and Workshop Pages - even though they are not Part of this years Conference ... but since we don't have active links to them that should be ok (and we will need this for the Archive later anyway)

Good point, let me check whether we can filter them by tag for now.

Update: Never mind, _not_ filtering them by tag allows us to send them a preview of their page before listing it.

**Pull request merged**